Makefile

```diff
@@ -70,8 +70,10 @@ CARGO_OSDK_ARGS += --grub-boot-protocol=$(BOOT_PROTOCOL)
 endif
 
 ifeq ($(ENABLE_KVM), 1)
+ifneq ($(INTEL_TDX), 1)
```

Comment:
Why can't KVM be enabled for TDX?

Reply:
Intel TDX currently only supports KVM. This configuration is directly reflected in the QEMU parameters of TDX:
`-accel kvm`

Reply:
Answer from GPT4:

Both --enable-kvm and -accel kvm are options used with QEMU, a hardware virtualization software, to accelerate virtual machine performance by using hardware virtualization extensions provided by modern CPUs. However, they are used in different versions or command syntaxes of QEMU.

--enable-kvm: This is the older syntax used in QEMU command-line options to enable KVM (Kernel-based Virtual Machine) acceleration. When you pass --enable-kvm to the QEMU command, it tells QEMU to use the KVM interface for virtualization, which allows it to leverage hardware extensions like Intel VT-x or AMD-V to run virtual machines much faster than purely emulated (software-based) virtualization.

-accel kvm: This is the newer syntax introduced in later versions of QEMU to specify the type of accelerator to use. The -accel option is more flexible and allows you to specify different acceleration backends. For example, -accel kvm tells QEMU to use KVM for acceleration, similar to --enable-kvm. It's a more explicit way to define the acceleration backend and is part of a more generalized syntax that can accommodate other types of accelerators as well, such as HAXM on Intel systems or HVF on macOS.

In summary, both options are used to achieve the same goal of enabling KVM acceleration, but -accel kvm is part of a newer, more flexible syntax that allows specifying different acceleration backends, while --enable-kvm is the older, more specific way to enable KVM. If you're using a recent version of QEMU, it's recommended to use the -accel option.

Reply:
So basically, they are the same. We should use one of them, not both. If the QEMU installed in our Docker images support is new enough to support `-accel kvm`, then we should use this recommended argument. If not, we should use `--enable-kvm` for both TDX and non-TDX settings.